semantics.json

```diff
@@ -85,6 +85,49 @@
           "type": "boolean",
           "label": "Correct",
           "importance": "low"
+        },
+        {
+          "name": "feedback",
```

Comment:
Added translation for all available languages (basically just copied a modified version from h5p-multi-choice). But: I only made some further clarification about the timing when the feedback is shown in the English and German translation as I don't speak the other languages. Although I don't think that the clarification is really needed (I just added it to the English and German version for convenience) I wonder what is the process for translating the semantic labels to all the languages? Does H5P has some tool for this? Otherwise it is quite cumbersome to add new fields to existing H5P content types.

Reply:
You'd use the English version if you cannot translate the language files yourself.

And yes, there's the [H5P CLI tool](https://h5p.org/h5p-cli-guide) that has a `create-language-file` command and an `update-translations` command.

If you add new fields like in this case, you'd add an attribute `"state": "new"` to the fields in `semantics.json` and running the `update translations` command would add that field to all the translation files (in English). Translations can (and for quite a bunch of languages will) be contributed very quickly by the people using https://translate-h5p.tk or manually.

Reply:
That makes sense and didn't know about Weblate. Will change it using the cli.